asconf/metadata/metadata.go

```diff
@@ -0,0 +1,53 @@
+package metadata
+
+import (
+	"fmt"
+	"regexp"
+	"sort"
+)
+
+var commentChar string
+var findComments *regexp.Regexp
+
+func init() {
+	commentChar = "#"
+	findComments = regexp.MustCompile(commentChar + `(?m)\s*(.+):\s*(.+)\s*$`)
```

Comment:
I assume we use the regex to make it easy to see what needs to be carried over. Maybe add an example of what this regex means. Not everything but at least `<key>: <val>`

Reply:
added a small example comment

Reply:
The first line would have sufficed haha